www/source/blog/2017-06-07-Triage-and-Contributions.html.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Triage and Contribution Process
+date: 2017-06-07
+author: Ian Henry
+tags: Governance,Community
+category: Community
+classes: body-article
+---
+A super common question that comes up in lots of FOSS communities is "How do I contribute". While most projects have a file like a [CONTRIBUTING.md](https://github.com/habitat-sh/habitat/blob/master/CONTRIBUTING.md) or instructions in a README on how to join in on the open source fun, this question still gets asked quite frequently. This is an interesting problem to have for a group of people that are actively attempting to grow their community and base of contributors. The question project maintainers really need to ask themselves is quite simply: Why is this question coming up in the first place?
+
+I've been told by folks several times over the last few months: "I'd love to get involved I just have no idea where to start." I've typically followed that question with a line about getting into the slack team and giving our [CONTRIBUTING.md](https://github.com/habitat-sh/habitat/blob/master/CONTRIBUTING.md) a read but, I think there is an inherent problem here that doesn't get discussed frequently/directly. It's not a problem specific to the habitat project but rather a side effect of collaborating on a thing with geo-spatially distributed actors. What I mean by that is: Collaborating over the internet is _hard_ and should be perpetually attempting to find ways to make that experience less painful.
+
+Many of us have probably watched other FOSS communities adopt different strategies to minimize the barrier to contribution and the habitat project is not an exception. One of our project beliefs is that building humane software requires our processes, our code of conduct, our Governance and our developer tooling to reflect that humanity. Which, is why we're opting to make some simple governance-ish changes that we hope to be the first steps towards minimizing the barrier to contribution in the [habitat.sh codebase](https://github.com/habitat-sh/habitat).
```

Comment:
This paragraph is a little repetitive with the last, you could probably collapse the two. If you don't, then you don't need the `,` after Which.